pom.xml

```diff
@@ -19,16 +19,18 @@
         <joda-time.version>2.7</joda-time.version>
         <logback.core.version>1.1.2</logback.core.version>
         <dozer.version>5.5.1</dozer.version>
+        <orika.version>1.5.0</orika.version>
```

Comment:
Hey Peter, it's definitely used. I agree that having multiple mapping frameworks is not ideal. However, I wouldn't shoot Orika down just because someone introduced Dozer before.

The Belgian team started with Dozer, but felt the need to switch. I'd like to discuss this outside this code review.